README.md

```diff
@@ -62,6 +59,13 @@ Install this project with:
 
 Node.js
 
+### Usage
+
+To run the project, execute the following command:
+
+- npm start
+- Fill in th inputs with your usernanme and score and submit
+- Click submit to see your score
 
```

Comment:
![suporting-section](https://user-images.githubusercontent.com/93039359/225331400-149016a8-2234-4d46-8f91-0d22da41b428.png)
- [ **OPTIONAL** ] **RECOMMENDED:** As mentioned by the previous reviewer, I think it would be better if you  you add the Show your **support** section in your readme file to make your readme descriptive and professional :blush: